source/geometry/FloatPoint3D.ooc

```diff
@@ -29,7 +30,8 @@ FloatPoint3D: cover {
 		if (r != 0.0f)
 			r = (this z / r) clamp(-1.0f, 1.0f) acos()
 		r
-	} }
+	}}
```

Comment:
Ah, we have both in the code. I see. Anyway, since we write `empty: static This { get { This new() } }`, there's a point in always having `} }` instead of `}}`. I'm not saying `} }` looks better though.